featuretools/entityset/entityset.py

```diff
@@ -280,231 +294,240 @@ def add_relationship(self,
             list[:class:`.Relationship`]: List of forward relationships.
         """
-        return [r for r in self.relationships if r.child_dataframe.id == entity_id]
+        return [r for r in self.relationships if r._child_dataframe_id == dataframe_id]
 
-    def get_backward_relationships(self, entity_id):
+    def get_backward_relationships(self, dataframe_id):
         """
-        get relationships where entity "entity_id" is the parent.
+        get relationships where dataframe "dataframe_id" is the parent.
 
         Args:
-            entity_id (str): Id of entity to get relationships for.
+            dataframe_id (str): Id of dataframe to get relationships for.
 
         Returns:
             list[:class:`.Relationship`]: list of backward relationships
         """
-        return [r for r in self.relationships if r.parent_dataframe.id == entity_id]
+        return [r for r in self.relationships if r._parent_dataframe_id == dataframe_id]
 
-    def has_unique_forward_path(self, start_entity_id, end_entity_id):
+    def has_unique_forward_path(self, start_dataframe_id, end_dataframe_id):
         """
         Is the forward path from start to end unique?
 
         This will raise if there is no such path.
         """
-        paths = self.find_forward_paths(start_entity_id, end_entity_id)
+        paths = self.find_forward_paths(start_dataframe_id, end_dataframe_id)
 
         next(paths)
         second_path = next(paths, None)
 
         return not second_path
 
     ###########################################################################
-    #  Entity creation methods  ##############################################
+    #  DataFrame creation methods  ##############################################
     ###########################################################################
 
-    def entity_from_dataframe(self,
-                              entity_id,
-                              dataframe,
-                              index=None,
-                              variable_types=None,
-                              make_index=False,
-                              time_index=None,
-                              secondary_time_index=None,
-                              already_sorted=False):
+    def add_dataframe(self,
+                      dataframe_id,
+                      dataframe,
+                      index=None,
+                      logical_types=None,
+                      semantic_tags=None,
+                      make_index=False,
+                      time_index=None,
+                      secondary_time_index=None,
+                      already_sorted=False):
         """
-        Load the data for a specified entity from a Pandas DataFrame.
+        Add a DataFrame to the EntitySet with Woodwork typing information.
 
         Args:
-            entity_id (str) : Unique id to associate with this entity.
+            dataframe_id (str) : Unique id to associate with this dataframe.
```

Comment:
since on the dataframe, this is going to be stored `df.ww.name` and `df.ww.id` doesn't exist, what if we changed the terminology to `dataframe_name`  (but still clarify the name should be unique in the docstring)?  I think it would help readability in other places when we have variables like `parent_datframe_id` - if we changed those to `parent_dataframe_name` i think it would be easier to intuit where that value came from

Reply:
yeah, that's a good idea! Were you also thinking we'd update some of the column terminology like `parent_column_id` to `parent_column_name`?

Reply:
yes, similar situation where `Variable.id` is now `ColumnAccessor.name`

Reply:
cool, I'll change all of those to use `name` instead of `id`

Reply:
updated dataframe and column id variables to use name instead. Left `EntitySet.id` since that felt separate enough, but happy to change if you think it should match.